FirelandsProject/cataclysm
# Core/Misc: Update locales_gossip_menu_option to simple system

### Changes proposed

- Update the gossip table, so that it contains the structure of the current emulators.

### Issues addressed


- Closes

### SOURCE:



### Tests performed


- Windows 10
- In-game

### How to Test the Changes:


1. Apply the change, and review the in-game gossip. If a link has been made to a translation and the translation is added, it should be visible in-game.

### Known issues and TODO list



---

- Thank you for collaborating with the project.
- For an efficient working methodology.
- It is obvious to make long pull request, unless it is necessary.
- A short pull request would be easy to test and approve.
